src/main/java/org/jenkins/tools/test/PluginCompatTesterCli.java

```diff
@@ -104,13 +106,17 @@ public class PluginCompatTesterCli implements Callable<Integer> {
     private String fallbackGitHubOrganization;
 
     @CheckForNull
-    @CommandLine.Option(names = "--mvn", description = "The path to the Maven executable.")
+    @CommandLine.Option(
+            names = "--mvn",
+            description = "The path to the Maven executable.",
+            converter = ExistingFileTypeConverter.class)
```

Comment:
arguably this is incorrect but we already call [`getAbosultePath`](https://github.com/jenkinsci/plugin-compat-tester/blob/943a678afd97f11f662203ebe33b30b7601163f9/src/main/java/org/jenkins/tools/test/maven/ExternalMavenRunner.java#L61) so we can not pass in something that we expect to be on the path.